core/blockchain.go

```diff
@@ -217,6 +217,9 @@ func NewBlockChain(db ethdb.Database, cacheConfig *CacheConfig, chainConfig *par
 	if cacheConfig == nil {
 		cacheConfig = defaultCacheConfig
 	}
+	if cacheConfig.TriesInMemory != 128 {
+		log.Warn("TriesInMemory isn't the default value(128) and may cause system instability", "triesInMemory", cacheConfig.TriesInMemory)
```

Comment:
Hi, can you change the warning info to `TriesInMemory isn't the default value(128), you need specify exact same TriesInMemory when prune data`